### PR TITLE
release-schema: Add Period.description

### PR DESCRIPTION
**Pull request by duncandewhurst**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ If items have at most one delivery address, use the [Location](https://extension
 
 ## Legal context
 
-In the European Union, this extension's fields correspond to [eForms BT-99 (Review Deadline Description), BT-163 (Concession Value Description), BT-109 (Framework Duration Justification), BT-505 (Organisation Internet Address), BT-508 (Buyer Profile URL) and BG-708 (Place of Performance)](https://docs.ted.europa.eu/eforms/latest/reference/business-terms/). For correspondences to Tenders Electronic Daily (TED), see [OCDS for the European Union](http://standard.open-contracting.org/profiles/eu/latest/en/).
+In the European Union, this extension's fields correspond to [eForms BT-99 (Review Deadline Description), BT-163 (Concession Value Description), BT-109 (Framework Duration Justification), BT-505 (Organisation Internet Address), BT-508 (Buyer Profile URL), BT-538 (Duration Other) and BG-708 (Place of Performance)](https://docs.ted.europa.eu/eforms/latest/reference/business-terms/). For correspondences to eForms fields, see [OCDS for eForms](http://standard.open-contracting.org/profiles/eforms/latest/en/). For correspondences to Tenders Electronic Daily (TED), see [OCDS for the European Union](http://standard.open-contracting.org/profiles/eu/latest/en/).
```
Comment (jpmckinney) [inline README.md:13]: #12 removes this section.

File: README.md [hunk 2]
```diff
@@ -111,6 +117,10 @@ Report issues for this extension in the [ocds-extensions repository](https://git
 
 ## Changelog
 
+### Unreleased
+
+* Add `Period.description`.
```
Comment (jpmckinney) [inline README.md]:
```suggestion
* Add `Period.description` field.
```

File: README.md [hunk 3]
```diff
@@ -111,6 +117,10 @@ Report issues for this extension in the [ocds-extensions repository](https://git
 
 ## Changelog
 
+### Unreleased
```
Comment (jpmckinney) [inline README.md]:
```suggestion
### 2023-06-26
```

File: README.md (hunk 1 above)
Comment (duncandewhurst) [reply README.md:13]: Ah, yes. I'll remove it when resolving merge conflicts, if that's OK?

Comment (jpmckinney) [reply README.md:13]: Sounds good

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**